src/web/components/Callout/Form.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { css } from 'emotion';
+
+import { calloutCampaign } from '@root/fixtures/calloutCampaign';
+import { Form } from './Form';
+
+export default {
+    component: Form,
+    title: 'Components/Callout/Form',
+};
+
+export const Default = () => {
+    return (
+        <div
+            className={css`
+                width: 630px;
+                padding: 15px;
+            `}
+        >
+            <Form
+                formFields={calloutCampaign.formFields}
+                submitForm={() => {}}
+                formId={1234}
+            />
+        </div>
+    );
+};
+Default.story = { name: 'default' };
```

Comment:
It's not clear what this test is doing. It's a nice pattern when the test data is either named to describe what we're testing or we use spreading to amend it in a way which describes it. When the data is hidden in an abstraction then there's no way to know the purpose of the test.

Reply:
I am not sure I follow, this is just rendering the component using fixtures, there isnt really any test (unless you are referring to visual regression?) 

Note: I am working on adding `Form.test.tsx` that will test this component